src/App.tsx

```diff
@@ -1,51 +1,69 @@
+import classNames from 'classnames';
 import React from 'react';
 import './App.scss';
 
-// import goodsFromServer from './goods';
+import goodsFromServer from './goods';
 
-export const App: React.FC = () => {
-  return (
-    <main className="App">
-      <header className="App__header">
-        <h1 className="App__title">
-          Jam is selected
-        </h1>
+export class App extends React.Component {
```

Comment:
Add types to the state, check [React + TS cheat sheet](https://mate-academy.github.io/fe-program/js/extra/react-typescript)